src/span_tween.rs

```diff
@@ -116,6 +116,7 @@ use std::{cmp::Ordering, ops, time::Duration};
 
 use crate::utils;
 use bevy::{ecs::system::EntityCommands, prelude::*};
+use bevy_eventlistener::prelude::*;
```

Comment:
The CI came in clutch. Needed a cfg here.

Reply:
Big whoops :sweat_smile:, fixed it.